cmd/geth/logtestcmd_active.go

```diff
@@ -42,7 +42,6 @@ This command is only meant for testing.
 // logTest is an entry point which spits out some logs. This is used by testing
 // to verify expected outputs
 func logTest(ctx *cli.Context) error {
-	log.ResetGlobalState()
```

Comment:
Note to self:  create and set a new default logger here to clear the field padding map.